### PR TITLE
Add pkolakaluri jersey city file

### LINKED ISSUE #24: Create a new file for githubteacher San Diego, CA
- [x] Create a branch
- [x] Add Commits
- [x] Open a PR
- [ ] Have a discussion
- [ ] Seek Approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by pkolakaluri**
close #24

### CODE REVIEW

File: pkolakaluri-JerseyCity.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#Jersey City , NJ
+
+###  Food recommandations
+-- City Tocas
```
Comment (polinava) [inline pkolakaluri-JerseyCity.md]: How good is the food here?

**Comment by polinava** [comment]
🐶

File: pkolakaluri-JerseyCity.md (hunk 1 above)
Comment (pkolakaluri) [reply pkolakaluri-JerseyCity.md]: good

**Review by polinava — COMMENTED** [review]
great change!

**Pull request merged**